mod_reforged/hooks/skills/actives/shieldwall.nut

```diff
@@ -18,6 +18,13 @@
 			text = ::Reforged.Mod.Tooltips.parseString("Grants immunity against [Hook Shield|Skill+rf_hook_shield_skill]")
 		});
 
+		tooltip.push({
+			id = 6,
+			type = "text",
+			icon = "ui/icons/special.png",
```

Comment:
In the shield.nut you give the very similar tooltip entry about this mechanic the icon `icon = "ui/icons/fatigue.png"`. I much prefer that over a generic `special` icon whenever possible

Reply:
In this particular case I think special is better than the fatigue icon because in the `shield.nut` it is an effect that brother is gaining because of building fatigue. So the fatigue icon (or maybe a melee defense icon) is more representative in that case. Whereas here this perk is removing the penalty which is already present - so the special icon (or a melee defense icon) is more representative of what this perk does. I avoided using the melee defense icon because that is already used in the tooltip of these things so I didn't want to spam it.